examples/alias/main.tf

```diff
@@ -2,7 +2,6 @@ provider "aws" {
   region = "eu-west-1"
 
   # Make it faster by skipping something
-  skip_get_ec2_platforms      = true
```

Comment:
removed due to 
![image](https://user-images.githubusercontent.com/10913471/229621757-6f7d3b5d-cd06-413d-ba74-99d8d97ae94f.png)